.storybook/main.js

```diff
@@ -8,12 +8,11 @@
  */
 module.exports = {
   stories: [
```

Comment:
Moving this stuff around so that we can make "Getting started" the page people see when they land on the root of the site instead of the more random tier1 color page